pom.xml

```diff
@@ -197,6 +283,11 @@ THE SOFTWARE.
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.kohsuke.metainf-services</groupId>
+      <artifactId>metainf-services</artifactId>
+      <optional>true</optional>
+    </dependency>
```

Comment:
Needed to create the `META-INF/services` index for the two `JettyProvider` implementations below.